CMakeLists.txt

```diff
@@ -13,7 +13,16 @@
 # limitations under the License.
 
 cmake_minimum_required(VERSION 3.8)
-project(mgclient VERSION 0.1)
+project(mgclient VERSION 1.1.0)
+
+# Minor version increase also means ABI incompatibility with previous versions,
+# because of that SO version could be calculated as a sum of major and minor
+# version numbers.
+math(EXPR mgclient_SOVERSION
+     "${mgclient_VERSION_MAJOR} + ${mgclient_VERSION_MINOR}")
+add_definitions(-DMGCLIENT_VERSION="${mgclient_VERSION}"
+                -DMGCLIENT_SOVERSION="${mgclient_SOVERSION}")
```

Comment:
`MGCLIENT_SOVERSION` is not needed.

Reply:
Agree. And isn't the statement above wrong? You can't rely on the sum of major and minor only, because 3.1 and 1.3 are the same then?

Reply:
Yep, my bad... I'll change to set SOVERSION manually...